### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio error axe-core Error:Cannot read property of undefined (reading, '_isDisabled') - feature unsupported in your environment. Skipping Color Contrast rule

### ISSUE THREAD

**Issue by jegadeessa** [GoogleChrome/lighthouse#14433]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).

### URL

Since this application is still in the development phase and this issue is happening in our lower environments (QA, UAT). Due to some restrictions, we can't share those URLs.

### What happened?

When we are trying to generate the LH report. It is throwing error for accessibility. PFB the attached screenshot.

<img width="1420" alt="Screenshot 2022-10-06 at 9 18 56 AM" src="https://user-images.githubusercontent.com/22210198/194210581-0b74ad5e-ba65-4082-8896-5432a66e14f9.png">

### What did you expect?

It should return the accessibility score.

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

9.6.2

### Chrome Version

106.0.5249.91 (Official Build) (64-bit)

### OS

Windows 10 Enterprise 

Version - 21H2

**Comment by jegadeessa** [comment]
@connorjclark Could you please help us with this issue? It's a kind of blocker for us.

**Comment by connorjclark** [comment]
Since you can't share a URL, can you share the JSON? Top right corner of the report there is a drop down menu.

**Comment by jegadeessa** [comment]
@connorjclark PFB the attached LHR JSON File.
[LH-Issue.json.zip](https://github.com/GoogleChrome/lighthouse/files/9734232/LH-Issue.json.zip)

**Comment by jegadeessa** [comment]
@connorjclark Could you please help us on this?

**Comment by connorjclark** [comment]
Lighthouse version 9.6.2 is the latest in stable chrome devtools, but we have tons of changes since then non released. Can you try with our latest nightly?

```
npx lighthouse@next https://uat.aem.anthemeap.com/manorhouse-management/connect-with-a-counselor --view
```

and share the same JSON if it happens again.

**Comment by brendankenny** [comment]
Closing as there hasn't been any update. Please comment if it's still a problem and we can re-open!
